Piwik.Tracker/PiwikTracker.cs

```diff
@@ -738,7 +759,7 @@ public TrackingResponse DoBulkTrack()
             }
 
             var postData = new JavaScriptSerializer().Serialize(data);
-            var response = SendRequest(PiwikBaseUrl, "POST", postData, true);
+            var response = await SendRequestAsync(PiwikBaseUrl, "POST", postData, true).ConfigureAwait(false);

```

Comment:
If we "await" here, is "BulkTrackAsync" really asynchronous?

My apologies if I am missing something.